ports/glfw3/portfile.cmake

```
@@ -6,8 +6,15 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
-if(VCPKG_TARGET_IS_LINUX)
-    message(
+if (VCPKG_TARGET_IS_EMSCRIPTEN)
+    # emscripten has built-in glfw3 library
+    set(VCPKG_BUILD_TYPE release)
+    file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/glfw3Config.cmake" DESTINATION "${CURRENT_PACKAGES_DIR}/share/glfw3")
+    set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+else()
```

1. Move this to the top.
2. ```suggestion
       return()
   endif()
   ```
3. Restore everything else in the top-level scope/indent.